### ISSUE TITLE
Transformations: Filter by data should be able to use numeric filters like greater than, less than and between for time fields

### ISSUE THREAD

**Issue by oscarkilhed** [grafana/grafana#76005]
**Why is this needed**:

It's not possible to properly filter data using transforms on time values, the only options are about null state or equal.

**What would you like to be added**:

Add the numeric filtering options to time fields.

**Comment by jHumeLPL** [comment]
yes, yes, yes, yes, yes, and **yes.**  Man, is there a way to vote for these feature requests?  I mean, I gave this a thumbs-up, but who knows if they use that to gauge popularity of requests.  

I also fear that feature requests are really hard to find and that a lot of people end up making the same feature requests as others but with different wording...  I diligently searched to find this one before I made one, but I'm sure there has to be others.

But I digress, yes, please Grafana devs, this _really_ needs to be a thing.  For me I need this ability in the **Filter by Values** Transformation, but same thing really.

I'm honestly kind of surprised that whoever built the **Filter by Data/Values** transformation didn't catch this as it seems pretty obvious that people would want to be able to do something like this...  

In my use case, the Dynatrace API has a horrible "feature" where, depending on where in the time slot the last value in a metric query falls, it may return a zero if it hasn't polled the data yet (I mean, at least return a null Dynatrace...).  Most API's would just not send the last timestamp if it hasn't polled a data, but not Dynatrace!  No, they think we want that fake zero for some reason....  Now, of course, you can use **Time Shift** with a value of **-1m** in the Grafana **Query Options** to "fix" the problem, but what this really does is shift _all_ the values by 1 minute, meaning the values in Grafana are actually 1 minute older than they really are.  Not the best fix.  

So, what I want to do is have a filter that says _Exclude a value if it is zero AND the time is greater than or equal to ${__to} - 1 minute_ (_meaning I need the ability to do math on the time field as well, which is a completely separate feature request of course._).
